src/megatron/bridge/models/conversion/auto_bridge.py

```diff
@@ -353,13 +358,36 @@ def export_hf_weights(
             conversion_tasks=conversion_tasks,
         )
 
+    def set_custom_modeling_source(self, source_path: Union[str, Path]) -> None:
```

Comment:
will it only be needed during checkpoint saving? if so, just add source path arg to saving api.